containers/Dockerfile.buildenv

```diff
@@ -1,10 +1,29 @@
 # Distributed under the MIT License.
 # See LICENSE.txt for details.
 
-# If you change this file please push a new image to DockerHub so that the
-# new image is used. Docker must be run as root on your machine. There are 3
-# different images in this Dockerfile. Here is how to build them for x86_64
-# (for Apple Silicon, replace linux/amd64 with linux/arm64):
+# If you change this file please push the new images to DockerHub. If you do not
+# have permission to push to DockerHub please coordinate with someone who does.
+# Since changes to this image affect our testing infrastructure it is important
+# all changes be carefully reviewed before they are pushed. This is how you use
+# GitHub Actions to build and push the new images:
+#
+# 1. Push your changes to the `develop` branch on your fork of the spectre
+#    repository on GitHub (https://github.com/YOURNAME/spectre).
```

Comment:
`YOURNAME` -> `GITHUBUSERNAME` ?

Reply:
I prefer the "your" in there

Reply:
`YOURGITHUBUSERNAME`? Because https://github.com/kylenelli/spectre doesn't exist (or at the very least, isn't me)

Reply:
I trust you find your GitHub account :)

Reply:
I don't mean to be provocative, I just think this is quite obvious and not worth a prolonged discussion